spk/gentoo-chroot/Makefile

```diff
@@ -0,0 +1,92 @@
+SPK_NAME = gentoo-chroot
+SPK_VERS = 13.0
+SPK_REV = 1
+SPK_ICON = src/gentoo.png
+DSM_UI_DIR = app
+
+DEPENDS = 
+WHEELS = pyextdirect==0.3.1 flask Werkzeug Jinja2 itsdangerous
+SPK_DEPENDS = "python>2.7.3-3"
+
+MAINTAINER = "SynoCommunity"
+DESCRIPTION = "Gentoo is a free operating system based on either Linux or FreeBSD that can be automatically optimized and customized for just about any application or need. Gentoo Chroot allows you to install the Gentoo OS inside your DiskStation, alongside DSM. This package is intended for advanced users only."
+RELOAD_UI = yes
+DISPLAY_NAME = "Gentoo Chroot"
+CHANGELOG = 
+
+HOMEPAGE   = "http://www.gentoo.org/"
+LICENSE    = 
+HELPURL    = 
```

Comment:
Drop this one, L20 covers it :) It won't do much, but just leave it in, until we get to fixing that.